backend/osmosewebsite/migrations/0022_alter_trap_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.23 on 2024-07-02 10:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("osmosewebsite", "0021_auto_20240702_1039"),
```

Comment:
The 0021_auto_20240702_1039 migration is missing

Reply:
I added them in the second commit